python/py-boltons/Portfile

```diff
@@ -0,0 +1,33 @@
+
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-boltons
+version             20.2.1
+platforms           darwin
+license             Apache
+supported_archs     noarch
+
+maintainers         {@harens gmail.com:harensdeveloper} \
+                    openmaintainer
+
+description         Constructs, recipes, and snippets which extend the Python standard library
+long_description    ${description}
+
+homepage            https://boltons.readthedocs.org/
+
+checksums           rmd160  26dbb3437f581226f1b3772b2c0d5bfddec4de04 \
+                    sha256  dd362291a460cc1e0c2e91cc6a60da3036ced77099b623112e8f833e6734bdc5 \
+                    size    172910
+
+python.versions     27 35 36 37 38
```

Comment:
There is no need for the older pythons. Just use version 38 to start and we will add new python versions as they become available.

Reply:
Sure thing, thanks.